exporters/ostream/src/metric_exporter.cc

```diff
@@ -1,68 +1,95 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#include <chrono>
 #ifndef ENABLE_METRICS_PREVIEW
-#  include "opentelemetry/exporters/ostream/metric_exporter.h"
 #  include <algorithm>
+#  include "opentelemetry/exporters/ostream/metric_exporter.h"
 #  include "opentelemetry/sdk/metrics/aggregation/default_aggregation.h"
 #  include "opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h"
 #  include "opentelemetry/sdk_config.h"
 
+namespace
+{
+std::string timeToString(opentelemetry::common::SystemTimestamp time_stamp)
+{
+  auto duration = time_stamp.time_since_epoch().count();
+
+  std::chrono::nanoseconds dur(duration);
+  std::chrono::time_point<std::chrono::system_clock> dt(dur);
```

Comment:
nit - need to check, but I think this can be directory obtained by calling `time_stamp()`?

Reply:
I try to print time in human readable format here. I didn't find any function similar.

Reply:
Not sure if something wrong with GitHub. I pasted a comment here, it was displayed multiple times and then when I removed one occurrence all of them removed :) But it was not that important, was thinking something to avoid couple of lines of code ( so can be ignored):

```cpp
std::string timeToString(opentelemetry::common::SystemTimestamp time_stamp)
{
  std::time_t epoch_time = std::chrono::system_clock::to_time_t(time_stamp());
  return std::ctime(&epoch_time);
}
```

Reply:
thanks, added your version.